nmagent/responses.go

```diff
@@ -36,3 +36,7 @@ type NCVersion struct {
 type NCVersionList struct {
 	Containers []NCVersion `json:"networkContainers"`
 }
+
+type HomeAzResponse struct {
+	HomeAz string `json:"HomeAz"`
```

Comment:
UINT

Reply:
@ashvindeodhar why uint?

Reply:
NMA returns this as UINT. We will be passing this back to NMA when we create NC, so keeping it UINT with us